src/tests.rs

```diff
@@ -56,11 +57,11 @@ fn tcp_connection() {
     assert_eq!(
         app_server
             .world
-            .resource::<Vec<ServerConnection<TcpConfig>>>()
+            .get_resource::<Events<NewConnectionEvent<TcpConfig>>>()
+            .unwrap()
             .len(),
         1,
-        "Vec<ServerConnection>'s length is not 1"
-    );
```

Comment:
Any reason why you changed it to Events? Btw it should actually be `ServerConnections` instead of `Vec<ServerConnection>`, it was added in #3, but wasn't changed here, nice catch. Not sure why tests didn't fail though

Reply:
Test was broken, I found it only when upgrade was done so fixed just to fix all :) Also tests module commented cause some CI problems (I get it in comments).

Reply:
The test was broken because it should be `crate::server::ServerConnections` instead of `Vec<ServerConnection>`. ServerConnections is a resource that is a Vec of all active clients connected to the server, that's what this test was supposed to test.